### PR TITLE
[chore] Bump sdk version in cli and local

### PR DESCRIPTION
**Pull request by avichalp**
## Summary

Updates the version of tableland/sdk package which uses the latest tableland/evm

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/local/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tableland/local",
-  "version": "2.2.0",
+  "version": "2.3.0",
```
Comment (joewagner) [inline packages/local/package.json:3]: I think this value was already updated in #81.  I think we want this to be 2.2.0

File: packages/cli/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tableland/cli",
-  "version": "5.3.2",
+  "version": "5.4.2",
```
Comment (joewagner) [inline packages/cli/package.json:3]: The next version here would be 5.4.0 right?

Comment (avichalp) [reply packages/cli/package.json:3]: oh yeah! updating

File: packages/local/package.json (hunk 1 above)
Comment (joewagner) [reply packages/local/package.json:3]: Sorry, this is a bit confusing.  The version of local has already been updated in #81 here:
https://github.com/tablelandnetwork/tableland-js/pull/81/files#diff-eebd2b35696842ddf3f1a4f7248077fb077628ca056a247012c7b24ca646254dR3

You can see that the current version pulbished to npm is 2.1.1 
https://www.npmjs.com/package/@tableland/local?activeTab=versions

Which means the next version to be published should be 2.2.0 right?

**Review by joewagner — APPROVED** [review]

File: packages/local/package.json (hunk 1 above)
Comment (avichalp) [reply packages/local/package.json]: ah... got it! the version was updated in #81 but not published. thanks for the explanation, fixed it.

**Pull request merged**